stdlib/types.pyi

```diff
@@ -287,7 +287,33 @@ class CodeType:
             co_filename: str = ...,
             co_name: str = ...,
             co_lnotab: bytes = ...,
-        ) -> CodeType: ...
+        ) -> Self: ...
+
+    if sys.version_info >= (3, 13):
+        # Same as replace, but positional for `self`.
```

Comment:
I don't think there's a real difference there, `self` is always positional-only for methods implemented in C.

Reply:
I saw this under `date`, so wasn't sure, let me simplify things then.

Reply:
https://github.com/python/typeshed/pull/12265/commits/d5c706afb2cd38e0613d506b9becec9fbc5fdd9f